docs/reference/data-streams/set-up-a-data-stream.asciidoc

```diff
@@ -154,6 +157,70 @@ PUT /_data_stream/my-data-stream
 // TEST[continued]
 // TEST[s/my-data-stream/my-data-stream-alt/]
 
+[discrete]
+[[convert-an-index-alias-to-a-data-stream]]
+=== Convert an index alias to a data stream
+
+Prior to {es} 7.9, you would typically use an <<indices-aliases,index alias>>
+with a write index to manage time series data. Data streams replace most of
+this functionality and usually require less maintenance.
+
+To convert an index alias with a write index to a new data stream with the same
+name, use the <<indices-migrate-to-data-stream,migrate to data stream API>>.
+During conversion, the alias’s indices become hidden backing indices for the
+stream. The alias’s write index becomes the stream’s write index. Note the data
+stream still requires a matching <<create-a-data-stream-template,index
+template>>.
+
```

Comment:
There are limitations on the kinds of indices that can be migrated to data streams, but I don't know whether it's important to mention those here. Mentioning them would essentially duplicate the content of the docs for the migration API.

Reply:
Agree. I think the xref to the API will suffice. If we get feedback to the contrary, I can reuse the reqs from the API docs here without duplicating the source. However, I'd like to avoid that if able.